arenadata/scripts/run_resgroup_test.bash

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+mkdir ssh_keys -p
+if [ ! -e "ssh_keys/id_rsa" ]
+then
+  ssh-keygen -P "" -f ssh_keys/id_rsa
+fi
+
+docker network create resgroup_net
+
+#start cdw and sdw1 containers
+docker run -d --rm \
```

Comment:
why not use docker compose like in behave tests?

Reply:
It is possible, but I thought that since we only need two containers, it would be superfluous.

Reply:
If you run tests for resource groups in the same way as behave tests through Docker Compose, then you do not need to modify the demo cluster and this modification can be done in a separate task, because it is still useful and necessary.

Reply:
Technically, it is possible to use containers for behave tests to run tests of resource groups but. We will need to put the initialization of containers in a separate script to avoid duplication. We still need the ability to deploy a demo cluster on multiple hosts.

Reply:
Don't behave tests run a cluster in multiple containers without using the new demo cluster feature?

Reply:
Yes, but they use it https://github.com/arenadata/gpdb/blob/adb-6.x/gpMgmt/test/behave_utils/cluster_setup.py

Reply:
Then, it turns out that with the new feature of the demo cluster it is possible to simplify the behave tests?

Reply:
Maybe. but do we want such big changes?

Reply:
If we want to use docker-compose for behave tests, then it's probably worth waiting for merge https://github.com/arenadata/gpdb/pull/885 to avoid conflicts.

Reply:
It was merged

Reply:
So in the end, you think it's worth reusing the containers of the behave tests? Right?

Reply:
> the containers of the behave tests

may be not of but like in